apps/stable_diffusion/src/utils/resources/model_db.json

```diff
@@ -1,6 +1,6 @@
 [
   {
-    "stablediffusion/untuned":"gs://shark_tank/stable_diffusion",
+    "stablediffusion/untuned":"gs://shark_tank/latest",
```

Comment:
can we pin this to a SHA from last night ? OR should we let it run free ? It will cause a download every day for the end user.

Reply:
what causes it to download every day? Shouldn't it check the hash to see if it has changed? Shouldn't unless the model changes

Reply:
Ok let's push and try. I thought the hash will change given they nightly update to the tank